### PR TITLE
fix use cli plugin

### PR DESCRIPTION
**Pull request by elevatebart**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,16 +10,12 @@
     "lint:css": "stylelint src",
     "test": "vue-cli-service test:unit --maxWorkers=1",
     "test-ci": "vue-cli-service test:unit --reporters=default --reporters=jest-junit --maxWorkers=1",
-    "docs": "vue-styleguidist server",
-    "docs:build": "vue-styleguidist build"
+    "docs": "vue-cli-service styleguidist server",
+    "docs:build": "vue-cli-service styleguidist build"
```
Comment (shadow7412) [inline package.json]: Sorry about the delay. The project now compiles - but running `npm run docs:build` now seems to run the server.

**Review by shadow7412 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (elevatebart) [reply package.json]: You are right; I did not read my own docs... hum

`styleguidist` does not take any extra arguments.

I should have used 

- `syleguidist server` -> `styleguidist`
- `styleguidist build` -> `styleguidist:build`

I pushed a fix to my branch.

Sry

**Review by shadow7412 — CHANGES_REQUESTED** [review]
Yep, that was the issue. I really appreciate your patience during this, and I'm hoping this is the last thing, but in my project, the configuration is being ignored.

If you do a build, you should see a welcoming page (as specified in 'sections') but instead it's just processing all components it can find, which I believe is the default behaviour.

Trying to build this version is giving me the following error too;
```
TypeError: Cannot read properties of null (reading 'useContext')
    at Ribbon (webpack://docsrepro/./node_modules/react-styleguidist/lib/client/rsg-components/Ribbon/Ribbon.js?:12:107)
    at div
    at StyleGuideRenderer (webpack://docsrepro/./node_modules/vue-styleguidist/lib/client/rsg-components/VsgStyleGuide/StyleGuideRenderer.js?:81:23)
    at _class (webpack://docsrepro/./node_modules/react-styleguidist/lib/client/rsg-components/Styled/Styled.js?:63:28)
    at StyleGuide (webpack://docsrepro/./node_modules/react-styleguidist/lib/client/rsg-components/StyleGuide/StyleGuide.js?:91:29)
This may be due to an error in a component you are overriding, or a bug in React Styleguidist.

If you believe this is a bug, [please submit an issue](https://github.com/styleguidist/react-styleguidist/issues).
```

**Comment by shadow7412** [comment]
@elevatebart for what it's worth, the way I was doing it is still the way the documentation says to do it (see https://vue-styleguidist.github.io/docs/GettingStarted.html#_1-install).

If this is wrong, would you consider updating the documentation?

**Pull request merged**